reader/processor/processor.go

```diff
@@ -112,8 +112,18 @@ func isAllowedEntry(feed *model.Feed, entry *model.Entry) bool {
 // ProcessEntryWebPage downloads the entry web page and apply rewrite rules.
 func ProcessEntryWebPage(feed *model.Feed, entry *model.Entry) error {
```

Comment:
This function is called only from the web ui. You probably want to have this new feature during feed refreshes as well.

Reply:
Added the code to function `ProcessFeedEntries` too.